src/converse-controlbox.js

```diff
@@ -610,8 +610,18 @@ converse.plugins.add('converse-controlbox', {
                     }
                 }
             });
-        });
 
+        });
+        _converse.api.listen.on('chatBoxViewPortCalculate', () => {
+            const vh = window.innerHeight * 0.01;
+            document.documentElement.style.setProperty('--vh', `${vh}px`);
+        });
+        _converse.api.listen.on('chatBoxViewPortCaclulateWhenResize', () => {
+            window.addEventListener('resize', () => {
+                const vh = window.innerHeight * 0.01;
+                document.documentElement.style.setProperty('--vh', `${vh}px`);
+            });
+        });
```

Comment:
Instead of making custom events, you can just listen for the `chatBoxViewsInitialized` event.